src/components/Attachment/FileAttachment.tsx

```diff
@@ -57,30 +57,6 @@ const styles = StyleSheet.create({
   },
 });
 
-export const getFileSizeDisplayText = (size?: number | string) => {
```

Comment:
Styleguidist (docgen) picks up first function in file, so need to move these to bottom of file.

Reply:
Gotcha, yeah another reason not to use it ;)

Reply:
We can separate out to a different file I guess!!

Reply:
all good for now